safe-global/safe-react
# Feat: add mobile banner on Dashboard

## What it solves
Part of https://github.com/safe-global/safe-pm/issues/54

## How this PR fixes it
Adds a new widget on the Dashboard. It's an interactive slideshow banner that you can close.

## How to test it

- Go to the dashboard
- Click through the slides in the banner
- Close the banner
- Choose an answer
- Check that the correct GTM event was sent (the two buttons send a diff event each)
- Reload the page – banner should not appear again

## Analytics changes
Sends a new customClick event with the category `mobile-app-promotion`.
The app download badge is tracked via a referral link.

## Todo
* [x] Request a new referral link from @DmitryBespalov
